jwala-webapp/src/main/webapp/resources/js/react/group-operations/GroupOperationsDataTable.js

```diff
@@ -815,12 +815,8 @@ var GroupOperationsDataTable = React.createClass({
     webServerConfirmDeleteCallback: function(data) {
         let self = this;
         webServerService.deleteWebServer(data.id.id, true).then(function(e){
-            // Remove deleted row
-             $("[id*='ws-child-table']").find("td:contains('" + data.name + "')").filter(function() {
-                if ($(this).text() === data.name) {
-                    $(this).parent().remove();
-                }
-            });
+            $.alert("Web server " + data.name + " was successfully deleted. Jwala will need to refresh to display the latest data and recompute the states.",
```

Comment:
There is, but I need to refactor it. Doing it with workarounds would take the same time in my calculations than just rewriting it taking advantage of React. React is perfect for this scenario but sad to say React is not mostly at play here.